.changeset/famous-dots-dance.md

```diff
@@ -0,0 +1,5 @@
+---
+'@lit-labs/context': patch
+---
+
+lower target js version
```

Comment:
These show up in the generated changelogs, so usually we want a little nicer presentation with capitalization and some details that make it understandable in that context like "TypeScript" and "JavaScript version". I'll take care of this post merging though!